script.js

```diff
@@ -50,17 +51,32 @@ function playerTurn() {
 
 function removeEventListener() {
     document.querySelectorAll('.box').forEach(element => {
-        element.removeEventListener('click',  boxClick)
+        element.removeEventListener('click', boxClick)
     })
 }
 
+function speedLvl() {
+    let minimumSpeed = 300;
+    if (speedTime === 300) {
+        speedTime = minimumSpeed;
+    } else if (level % 4 === 0) {
+        speedTime -= 100;
+    }
+}
+
 function boxClick(e) {
     if (e.target.id !== colorPattern[0]) {
         toggleEndGameModal()
     } else {
         colorPattern.shift()
         if (colorPattern.length === 0) {
             timesRun = 1
+            level++
+            if (level !== 1 && level %2 !== 0) {
```

Comment:
this feels like it should be its own function.

also missing a space before the `%`